packages/sdks/snippets/angular-16/src/app/live-preview/live-preview.component.ts

```diff
@@ -0,0 +1,68 @@
+import { CommonModule } from '@angular/common';
+import type { OnDestroy, OnInit } from '@angular/core';
+import { Component, inject } from '@angular/core';
+import { Router } from '@angular/router';
+import type { BuilderContent } from '@builder.io/sdk-angular';
+import { fetchOneEntry, subscribeToEditor } from '@builder.io/sdk-angular';
+@Component({
+  standalone: true,
+  selector: 'app-live-preview',
+  imports: [CommonModule],
+  template: `<div *ngIf="loading">Loading Data...</div>
+    <div *ngIf="!loading && content" class="blog-data-preview">
+      <div>Blog Title: {{ content.data?.title }}</div>
+      <div>Authored by: {{ content.data?.['author'] }}</div>
+      <div>Handle: {{ content.data?.['handle'] }}</div>
+      <div>
+        Published date:
+        {{
+          content.data?.['publishedDate']?.split(' ')?.slice(0, 4)?.join(' ')
```

Comment:
This would be a lot cleaner as 
```suggestion
          content.data?.['publishedDate']
            ? new Date(content.data.publishedDate).toDateString() 
            : null
```

see 
![CleanShot 2025-02-19 at 09 59 02@2x](https://github.com/user-attachments/assets/988abb7b-079d-49e3-a884-740e6d096bfe)


I checked the React snippets, and this feedback also applies there

Reply:
Angular templating does not allow the use of new Date(). Therefore, I created a new function that provides the same functionality.